src/components/headers/TripHeader.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { View } from 'react-native'
+import IconButton from '../pressables/IconButton'
```

Comment:
Can we use the `@` alias always instead of relative paths?

`@/components/pressables/IconButton`